src/fixml/modules/workflow/parse.py

```diff
@@ -91,9 +90,14 @@ def get_completeness_score(self, score_format: str = 'fraction', verbose: bool =
                 print()
                 print(f"WARNING: `n_files_tested` is not unique!")
                 print()
-            print(f'Score: {score}')
+            print(f'Result: {num_items_satisfied} items are satisfied out of {num_items}')
+            print(f'Score: {fraction_satisfied*100:.1f}%')
```

Comment:
Python f-strings support float to percentage conversion, so it can be simplifed into something like `{fraction_satisfied:.1%}`

See:
- https://docs.python.org/3/library/string.html#format-specification-mini-language
- https://stackoverflow.com/questions/56410671/converting-from-float-to-percentage

Reply:
@SoloSynth1 done, thanks!